pkg/services/dashboards/models.go

```diff
@@ -375,6 +375,7 @@ type CountDashboardsInFolderQuery struct {
 // to the store layer. The FolderID will be replaced with FolderUID when
 // dashboards are updated with parent folder UIDs.
 type CountDashboardsInFolderRequest struct {
+	// Deprecated: use FolderUID instead
```

Comment:
Nit: might be helpful to change the phrasing to `use folder UID instead`. To me `FolderUID` sounds like the name of a field but `FolderUID` doesn't exist as a field. What do you think?